Upstream-Status: Pending

binutils build fails on Fedora18+ due to over array bounds issue: 
binutils-2.23.1/opcodes/rl78-dis.c:230:13: error: array subscript is above array bounds [-Werror=array-bounds]
      if (oper->use_es && indirect_type (oper->type))
              ^

Signed-off-by: Zhenhua Luo <zhenhua.luo@freescale.com>

--- binutils-2.23.1/opcodes/rl78-dis.c.orig	2013-03-12 22:17:47.664361066 -0500
+++ binutils-2.23.1/opcodes/rl78-dis.c	2013-03-12 23:39:51.383460914 -0500
@@ -221,7 +221,7 @@
 
 	    case '0':
 	    case '1':
-	      oper = opcode.op + *s - '0';
+	      oper = &opcode.op[*s - '0'];
 	      if (do_bang)
 		PC ('!');
 
